modules/ROOT/pages/mule-upgrade-tool.adoc

```diff
@@ -0,0 +1,135 @@
+= Mule Upgrade Tool
+
+*Latest version: 1.0.0*
+
+include::release-notes::partial$mule-upgrade-tool/mule-upgrade-tool.adoc[tag=intro]
+
+== Compatibility
+
+* The Mule upgrade tool runs in any environment that is capable of running a standalone Mule instance.
+* You can update only standalone Mule instances with this tool.
+* The upgrade tool is compatible with Mule 4 versions only.
+
+== Before You Begin
+
+Before getting started, ensure you have:
+
+* A Mule 4 instance currently configured and working in your local environment.
```

Comment:
I would move the last bullet to follow this one, since they both address your current installation:

* The Mule upgrade tool inside your current Mule instance's `$MULE_HOME/tools` directory.

If the Mule upgrade tool is not bundled with your current Mule distribution, download the tool from the https://help.mulesoft.com/s/[Support Portal^] and copy it into the `$MULE_HOME/tools` directory of your current Mule instance.